### DISCUSSION TITLE
Lesson 6, Part 16 - source .env not working

### DISCUSSION THREAD

**Discussion by 0xwtfwtf** [Cyfrin/foundry-full-course-cu#460] [General]
I created a .env file under /test with my private key and rpc-url and tried to use source .env in the terminal but it says no such file or directory is found. Is there code I need to add to some settings file to get this to work?

**Comment by palmcivet7** [comment]
If the `.env` file is in your `/test` directory, you will need to `cd test` for `source .env` to work. Otherwise move the `.env` file to your root directory, assuming that's where you're inputting the `source .env` command.

**Reply by satoshimasterbilly** [reply]
Thanks  for your advice and code took me 20 mins doodling 👿-worked a treat now and thanks again🤪

**Reply by thurendous** [reply]
This is an awesome answer and helped me! Thanks.
